src/Sarif.Sdk.sln

```diff
@@ -31,6 +31,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PREfastXmlSarifConverter",
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sarif.ValidationTests", "Sarif.ValidationTests\Sarif.ValidationTests.csproj", "{A35E38C4-8189-43C0-BA09-8D05B8EF0791}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SarifTests", "SarifTests\SarifTests.csproj", "{335D24C1-6242-4328-93B5-25E05AC90C89}"
+EndProject
```

Comment:
There is already a project `Sarif.UnitTests`. Please add your new tests to that project.